content/posts/plugin-updates.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Plugin Repositories"
+subtitle = "Install Plugins from Anywhere!"
+author = "dkanada"
+githubusername = "dkanada"
+date = "2020-07-17"
+justify = "center"
++++
+
+There have been several changes to plugins for the new release.
+Here's a quick post going over the biggest updates for developers and users alike.
+<!--more-->
+The most noticable change for everyone is that third party plugin repositories are now available!
+You can access the current list at Dashboard -> Plugins -> Repositories, and items can be added or removed as desired.
+Even the official repository can be removed if you'd prefer to avoid external calls to our server.
+
+# Manifest
+
+For developers, the only required change is a JSON manifest at *any* location with versions that point to binary releases at *any* location.
+We don't require any specific method for hosting these files, as that would go against the ideals of the project.
+The official repository uses nginx for the manifest and GitHub Releases for the binaries, but you could potentially host the manifest on GitHub as well.
+Here's an example manifest with all the properties.
+Please note that the GUID must be unique (both in the manifest and the plugin itself) if you want to avoid conflicts with other plugins.
+
+```json
+[
+    {
+        "category": "Metadata",
+        "guid": "a4df60c5-6ab4-412a-8f79-2cab93fb2bc5",
+        "name": "Anime",
+        "description": "This plugin supports several different metadata providers and options for organizing your collection.",
+        "owner": "jellyfin",
+        "overview": "Manage Your Anime in Jellyfin",
+        "versions": [
+            {
+                "checksum": "ad6db5175f4732308b5dd166f79a1c2d",
+                "changelog": "bug fixes and improvements",
+                "targetAbi": "10.6.0.0",
+                "sourceUrl": "https://repo.jellyfin.org/releases/other/whats-this-plugin.zip.gif",
+                "timestamp": "2020-03-27 06:02:58",
+                "version": "1.0.3.0"
+            }
+        ]
+    }
+]
+```
+
+The official repository manifest can be found <a href="https://repo.jellyfin.org/releases/plugin/manifest-stable.json">here</a> if you'd like more examples.
+You can also find the deprecated manifest at <a href="https://repo.jellyfin.org/releases/plugin/manifest.json">this link</a> for the old format.
```

Comment:
«here» and «this link» are poor choice of link text for accessibility reasons (screen readers)
[What’s wrong with using "Click Here" links?](https://webaccess.berkeley.edu/ask-pecan/click-here)